CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ Changes since the last release
 
 -   When generating cvmfsexec distribution for el9 machine type on an el7 machine, the factory reconfig and/or upgrade fails as a result of an error in `create_cvmfsexec_distros.sh`. This is possibly due to the tools for el7 being unable to handle el9 files (as per Dave Dykstra). Please exercise caution if using `rhel9-x86_64` in the `mtypes` list for the `cvmfsexec_distro` tag in factory configuration.
     -   Our workaround: the el9 machine type has been removed from the default list of machine types supported by the custom distros creation script (PR #312)
+-   Installing the factory on Alma9 results in missing monitoring and glidefactoryclient classad. As a consequence of this, the Factory continued to submit pilots since it was not aware of the submitted pilots. See Issue #338 for the technical details.
+    -   Our workaround: added a check to test if the rrd file under question exists. If the file is non-existent, fetching the rrd file will be skipped.
```

Comment:
Instead of these known issues add only a Bug Fix
```
-  Fixed fetch_rrd crash in EL9 causing  missing monitoring and glidefactoryclient classad (Issue #339, PR #339)
```
You can add the details here in the PR or Issue

Reply:
Updated as a bug fix instead of being a known issue as suggested. Will add more details in the PR/Issue shortly, including the test script that was used as part of the troubleshooting work.